app/controllers/home_controller.rb

```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+    allow_unauthenticated_access  # 👈 인증 없이 접근 허용!
+
+    def index
+      @hot_bills = Bill.order(proposed_at: :desc).limit(6)
```

Comment:
```suggestion
      @hot_bills = Bill.includes(proposals: :specific_proposer).order(proposed_at: :desc).limit(6)
```

요 부분은 어차피 나중에 바꾸긴 하겠지만 그래도 눈에 보여서 코멘트 달아두겠습니다 (N+1 쿼리 방지)

Reply:
이거는 그냥 제가 웹에서 커밋할게요.